packages/components/steps/src/index.ts

```diff
@@ -0,0 +1,48 @@
+import { CHANGE_EVENT } from '@element-plus/constants'
+import { buildProps, isNumber } from '@element-plus/utils'
+
+import type { ExtractPropTypes } from 'vue'
+
+export const stepsProps = buildProps({
+  space: {
+    type: [Number, String],
+    default: '',
+  },
+  active: {
+    type: Number,
+    default: 0,
+  },
+  direction: {
+    type: String,
+    default: 'horizontal',
+    validator: (val: string): boolean =>
+      ['horizontal', 'vertical'].includes(val),
+  },
+  alignCenter: {
+    type: Boolean,
+    default: false,
+  },
+  simple: {
+    type: Boolean,
+    default: false,
+  },
+  finishStatus: {
+    type: String,
+    default: 'finish',
+    validator: (val: string): boolean =>
+      ['wait', 'process', 'finish', 'error', 'success'].includes(val),
+  },
+  processStatus: {
+    type: String,
+    default: 'process',
+    validator: (val: string): boolean =>
+      ['wait', 'process', 'finish', 'error', 'success'].includes(val),
```

Comment:
Same here.

Maybe we can use a declaration for this:


```ts
const states = ['wait', 'process', 'finish', 'error', 'success'] as const
```

Reply:
Thank you! your Opinion!

Reply:
@JeremyWuuuuu Hello~!  Thank you always for your good reviews. I have reflected your review!